src/System.Reflection.Metadata/tests/TestUtilities/LoaderUtilities.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System.IO;
+using System.Reflection.PortableExecutable;
+using Xunit;
+
+namespace System.Reflection.Metadata.Tests
+{
+    internal unsafe static class LoaderUtilities
+    {
+        public static void LoadPEAndValidate(byte[] peImage, Action<PEReader> validator)
```

Comment:
So what happens to IL-only images? Are they mapped to memory as they are on disk without any relocations, etc.?